src/test/java/com/codejoust/main/util/MockHelper.java

```diff
@@ -15,7 +16,8 @@
 public class MockHelper {
 
     public static <T> T getRequest(MockMvc mockMvc, String url, Class<T> c, HttpStatus status) throws Exception {
-        MvcResult result = mockMvc.perform(get(url))
+        MvcResult result = mockMvc.perform(get(url)
+                .header(HttpHeaders.AUTHORIZATION, TestFields.TOKEN))
```

Comment:
Are there other `GET` requests in which the Header is not necessary, but is simply added to all the mocks and tests because if it isn't necessary, it doesn't hurt to include it?

Reply:
Yeah pretty much - lmk if you prefer an alternative at all though

Reply:
I think that's fine, just wanted to confirm that I understood the setup 👍